src/components/Basket/BasketCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Phones } from '../../types/phones.ts';
+import closeIcon from 'assets/close-grey.svg';
+import minusIcon from 'assets/minus.svg';
+import plusIcon from 'assets/plus.svg';
+import { useAppDispatch } from '../../hooks/hooks.ts';
+import { actions as basketActions } from '../../store/basketSlice.ts';
+import { Link } from 'react-router-dom';
+
+type Props = {
+  phone: Phones;
+};
+
+type Options = 'plus' | 'minus';
```

Comment:
here we can use enum like this: 
`enum Options {
    PLUS = 'plus',
    MINUS = 'minus'
}`